Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -101,6 +104,8 @@ class ShaderCode : public ShaderGeneratorInterface
 public:
   ShaderCode() { m_buffer.reserve(16384); }
   const std::string& GetBuffer() const { return m_buffer; }
+
+  // Deprecated: Writes format strings using traditional printf format strings.
```

Comment:
Can you mark this as `[[deprecated]]`? Or will it fail compilations due to "treat warnings as errors"?

Reply:
Compilation will fail.